practice/src/scripts/constants/notify.ts

```diff
@@ -12,63 +12,73 @@ export const Notify = {
     EDIT: "edit",
 };
 
-export const messageNotify: {
-    [x: string]: {
-        type: string;
-        message: string;
-    };
-} = {
-    [Notify.All]: {
-        type: Notify.All,
-        message: "Your action has been executed! The active tasks are showing.",
-    },
-    [Notify.ACTIVE]: {
-        type: Notify.ACTIVE,
-        message: "Your action has been executed! The active tasks are showing.",
+const MESSAGES = {
+    All : "Your action has been executed! The active tasks are showing",
```

Comment:
please find a meaningful word instead of ALL, I dont know ALL Which case does it represent?